install/environment.dev.yml

```diff
@@ -6,8 +6,5 @@ channels:
 dependencies:
```

Comment:
Since line [1] was not changed on this branch I cannot comment there.  
But I would like to.  
I think the name should not be the same as the "real" zarp envs (root/nonroot). maybe just `zarp-dev`?

Reply:
I thought the idea is that we update zarp with this one so should we not have the same name?

Reply:
Yes, this file is meant to _update_ the environment created with either `environment.yml` or `environment.root.yml` as per the installation instructions. If you think about it, it makes sense, too. Whatever is in `environment.yml`/`environment.root.yml` is what is required for ZARP to run. Whatever is in `environment.dev.yml` is what is needed in addition to run tests and develop. They are _not_ two separate environment, `environment.dev.yml` is an _extension_ of the `zarp` environment.